zach_brady/class6/animals/lib/animal.rb

```diff
@@ -0,0 +1,9 @@
+class Animal
+    attr_accessor :num_legs
+
+    @@num_legs = 4
```

Comment:
this is helpful, but I will need to reconstruct some of the issues I was having to see if this recreates them.